tools/docker-compose/Dockerfile

```diff
@@ -6,14 +6,20 @@ RUN yum -y update && yum -y install epel-release
 
 # sync with installer/roles/image_build/templates/Dockerfile.j2
 RUN yum -y install acl \
+  alsa-lib \
```

Comment:
Adding our UI deps to our devel image to simplify our environments. Since we take advantage of caching, this shouldn't be too painful. Refactoring things out into smaller components shouldn't be a separate task in the future.